examples/starknet-indexer/src/indexer.ts

```diff
@@ -19,21 +37,23 @@ export function createIndexerConfig(streamUrl: string) {
         },
       ],
     },
-    async transform({ block: { header, events, transactions, receipts } }) {
-      const ts = header?.timestamp!;
-      for (const event of events) {
-        // Use helpers to access transaction and receipt
-        const tx = getTransaction(event.transactionIndex!, transactions ?? []);
-        const receipt = getReceipt(event.transactionIndex!, receipts ?? []);
+    sink,
+    async transform({ block: { header }, context }) {
+      consola.info("Transforming block ", header?.blockNumber);
+
+      const { db, tables } = useSink({ context }) || {};
```

Comment:
I realized the drizzle sink doesn't need to return the `tables` since the user can just use the values they imported from their db (e.g. the `users` variable in this case). That should simplify the type _a lot_.